tests/README

```diff
@@ -0,0 +1,18 @@
+Test262
+=======
+
+As the official Test262 adds new tests now and then. We compressed a tar file
+which contains the snapshot of test262 and the test harness used in the paper.
```

Comment:
Should this be specific about the Onward! paper, since we also have the DLS 2012 paper that uses this repo?